test/WearableUIGallery/WearableUIGallery.Tizen.Wearable/WearableUIGallery.Tizen.Wearable.csproj

```diff
@@ -17,13 +17,17 @@
     <Folder Include="lib\" />
   </ItemGroup>
 
-  
   <!-- If solution already has PCL project, will reference -->
   <ItemGroup>
     <ProjectReference Include="..\..\..\src\Tizen.Wearable.CircularUI.Forms.Renderer\Tizen.Wearable.CircularUI.Forms.Renderer.csproj" />
     <ProjectReference Include="..\WearableUIGallery\WearableUIGallery.csproj" />
   </ItemGroup>
-  
```

Comment:
Should every user add this code in theirs app ?  is there any way to include edj file automatically in their's app.

Reply:
no. this is our test application referencing Circuliar UI project directly. In normal case(using pacakge reference), you don;t need to do anything.